odoo/addons/base/models/ir_ui_view.py

```diff
@@ -2700,6 +2700,29 @@ def get_formview_action(self, access_uid=None):
             'context': dict(self._context),
         }
 
+    @api.model
+    def get_records_action(self, records, **kwargs):
```

Comment:
There is no need for this to be public?  That way, I find it to have less value in base.

Reply:
good catch, you're right it can be private, js does not call it directly